pandas/plotting/_core.py

```diff
@@ -597,103 +397,227 @@ def boxplot_frame_groupby(grouped, subplots=True, column=None, fontsize=None,
         sharey=sharey, **kwds)
 
 
-# kinds supported by both dataframe and series
-_common_kinds = ['line', 'bar', 'barh',
-                 'kde', 'density', 'area', 'hist', 'box']
-# kinds supported by dataframe
-_dataframe_kinds = ['scatter', 'hexbin']
-# kinds supported only by series or dataframe single column
-_series_kinds = ['pie']
-_all_kinds = _common_kinds + _dataframe_kinds + _series_kinds
+class PlotAccessor(PandasObject):
+    """
+    Make plots of Series or DataFrame using the backend specified by the
+    option ``plotting.backend``. By default, matplotlib is used.
 
+    *New in version 0.17.0:* Each plot kind has a corresponding method on
```

Comment:
should fix these types of coments (and user versionadded and such); though this particular one can drop